bot/admin/admin.py

```diff
@@ -133,6 +133,85 @@ async def embed_available_cogs(self, ctx: commands.Context):
 
         await ch_bot.send(embed=embed)
 
+    @cmd_for_bot_stuff.command(name='load', hidden=True)
+    @command_log
+    async def load_extension(self, ctx: commands.Context, extn_name: str):
+        """Command handler for the `bot` subcommand `load`.
+
+        Loads an extension (Cog) with the specified name into the bot.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context from which this command is invoked.
+            extn_name (str): The name of the extension (Cog).
+        """
+        extn_name = extn_name.capitalize()
+
+        if "Cog" and "cog" not in extn_name:
```

Comment:
I guess the name should end with cog? 
In that case use function  [endswith() ](https://www.tutorialspoint.com/python3/string_endswith.htm)